src/tests/api/test_api_questions.py

```diff
@@ -29,7 +29,8 @@ def test_question_serializer(answer):
         "id",
         "variant",
         "question",
-        "required",
+        "question_required",
```

Comment:
Could you add these changes to the API documentation, please?

Reply:
API documentation is ready.